layouts/community/ortho_4x12/bredfield/keymap.c

```diff
@@ -155,10 +149,10 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
  * `-----------------------------------------------------------------------------------------------------------'
  */
 [_UI_LAYER] = LAYOUT_ortho_4x12(
-    GUI_TAB, ___X___, ___X___, ___X___, ___X___, ___X___, ___X___, ___X___, ___X___, GUI_MIN, GUI_EQL, ___X___,
-    GUI_GRV, GUI_1,   GUI_2,   GUI_3,   GUI_4,   GUI_5,   GUI_6,   GUI_7,   ___X___, GUI_LBR, GUI_RBR, ___X___,
-    _______, GUI_SSF, GUI_SST, ___X___, ___X___, ___X___, ___X___, ___X___, ___X___, ___X___, ___X___, ___X___,
-    _______, KC_MRWD, KC_MPLY, KC_MFFD, KC_BRID, _______, _______, KC_BRIU, KC_MUTE, KC_VOLD, KC_VOLU, _______
+    GUI_TAB, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, GUI_MIN, GUI_EQL, XXXXXXX,
+    GUI_GRV, GUI_1,   GUI_2,   GUI_3,   GUI_4,   GUI_5,   GUI_6,   GUI_7,   XXXXXXX, GUI_LBR, GUI_RBR, XXXXXXX,
+    _______, GUI_SSF, GUI_SST, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+    _______, KC_MRWD, KC_MPLY, KC_MFFD, KC_VOLD, KC_MUTE, KC_MUTE, KC_VOLU, _______, _______, _______, _______
```

Comment:
You removed your brightness control keys here, and I just wanted to make sure that was deliberate.

Everything else looks good.

Reply:
Yep, wasn't working with Mac OS. Thanks for checking

Reply:
IIRC, that's because the hardware ID ***must*** match to a known apple keyboard.